pkg/recipe/variable.go

```diff
@@ -350,6 +350,17 @@ func (vv *VariableValues) UnmarshalYAML(unmarshal func(interface{}) error) error
 					Rows:    rows,
 				}
 			}
+		// Multiselect values without a value are an empty []interface{} and we want to cast them into MultiSelectValues
+		case []interface{}:
+			multiV := make(MultiSelectValue, 0, len(v))
+			for _, interfaceV := range v {
```

Comment:
Minor thing, but since every element in original slice needs to map to the new one, we can set the elements to the new slice with indices instead of using `append`. So setting `multiV := make(MultiSelectValue, len(v))`, using `for i, interfaceV ...` and finally `multiV[i] = str`

Reply:
Good point!